### PR TITLE
Release updates md

### PR DESCRIPTION
**Pull request by mdeceglie**
- [x] ~Code changes are covered by tests~
- [x] ~New functions added to `__init__.py`~
- [x] ~API.rst is up to date, along with other sphinx docs pages~
- [x] Example notebooks are rerun and differences in results scrutinized
- [x] ~Updated changelog~

### CODE REVIEW

**Comment by mdeceglie** [comment]
See #211, #210,

File: rdtools/plotting.py [hunk 1]
```diff
@@ -340,7 +340,7 @@ def availability_summary_plots(power_system, power_subsystem, loss_total,
     ax3.set_ylabel('Estimated lost power [kW]')
 
     # cumulative energy
-    ax4.plot(energy_cumulative, label='Reported Production')
+    energy_cumulative.plot(ax=ax4, label='Reported Production')
```
Comment (kanderso-nrel) [inline rdtools/plotting.py:343]: For future readers:  this change has something to do with newer versions of pandas (e.g. 1.1.0) converting datetimes in an incompatible way with matplotlib, so mixing `data.plot(ax=ax)` with `ax.plot(data)` resulted in bad behavior.

**Review by kanderso-nrel — COMMENTED** [review]
I created a scratch py3.7 conda env, installed requirements.txt, and then notebook_requirements.txt, and no errors :+1:

Only issue I see is that I think one of the plotting cells in the availability notebook got duplicated.

**Comment by mdeceglie** [comment]
Good catch on the plot @kanderso-nrel. That has been removed. Thanks!

**Review by kanderso-nrel — APPROVED** [review]
LGTM!

**Pull request merged**